src/handshake/server.rs

```diff
@@ -30,7 +30,7 @@ pub type Request = HttpRequest<()>;
 pub type Response = HttpResponse<()>;
 
 /// Server error response type.
-pub type ErrorResponse = HttpResponse<Option<String>>;
+pub type ErrorResponse = HttpResponse<Vec<u8>>;
```

Comment:
I think this one (on the server side) should be kept as `Option<String>`.

Rationale: `ErrorResponse` in this context is also a response that the user returns from their own callback that is supplied.